package.json

```diff
@@ -5,9 +5,10 @@
   "description": "Electron application boilerplate based on React, React Router, Webpack, React Hot Loader for rapid application development",
   "main": "main.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 mocha --retries 2 --compilers js:babel-register --recursive --require ./test/setup.js test/**/*.spec.js",
+    "test": "cross-env NODE_ENV=test jest test/**/*.spec.js",
```

Comment:
We should keep `BABEL_DISABLE_CACHE=1` usage.

([babel-plugin-webpack-loaders/src/plugin.js#L208](https://github.com/istarkov/babel-plugin-webpack-loaders/blob/master/src/plugin.js#L208))